nengo_dl/tests/test_neurons.py

```diff
@@ -176,3 +183,75 @@ def test_spiking_swap(Simulator, rate, spiking, seed):
 
     # check that the gradients match
     assert all(np.allclose(g0, g1) for g0, g1 in zip(*grads))
+
+
+@pytest.mark.parametrize("Neurons", (LeakyReLU, SpikingLeakyReLU))
+def test_leaky_relu(Simulator, Neurons):
+    assert np.allclose(Neurons(negative_slope=0.1).rates([-2, 2], 1, 0), [[-0.2], [2]])
+
+    assert np.allclose(
+        Neurons(negative_slope=0.1, amplitude=0.1).rates([-2, 2], 1, 0),
+        [[-0.02], [0.2]],
+    )
+
+    with nengo.Network() as net:
+        vals = np.linspace(-400, 400, 10)
+        ens0 = nengo.Ensemble(
+            10,
+            1,
+            neuron_type=Neurons(negative_slope=0.1, amplitude=2),
+            gain=nengo.dists.Choice([1]),
+            bias=vals,
+        )
+        ens1 = nengo.Ensemble(
+            10,
+            1,
+            neuron_type=Neurons(negative_slope=0.5),
+            gain=nengo.dists.Choice([1]),
+            bias=vals,
+        )
+        p0 = nengo.Probe(ens0.neurons)
+        p1 = nengo.Probe(ens1.neurons)
+
+    with Simulator(net) as sim:
+        # make sure that ops have been merged
+        assert (
+            len(
+                [
+                    ops
+                    for ops in sim.tensor_graph.plan
+                    if isinstance(ops[0], nengo.builder.neurons.SimNeurons)
+                ]
+            )
+            == 1
+        )
+
+        sim.run(1.0)
+
+        assert np.allclose(
```

Comment:
Should these use the `allclose` fixture or do they intentionally use `np.allclose` to avoid being in the RMSE report?

Reply:
Nengo DL doesn't actually use the `allclose` fixture anywhere (no reason not to, just never got around to making the switch). So could add it here, or we could just do some pass in the future and switch everything over. Either one seems fine.

Reply:
Ah okay, I saw it in `.nengobones.yml` and assumed it was used. I'm cool leaving it as is for now.